k-distribution/tests/regression-new/no-pattern/Makefile

```diff
@@ -0,0 +1,8 @@
+DEF=test
+EXT=test
+TESTDIR=.
+KOMPILE_BACKEND=llvm
+KOMPILE_FLAGS+=--enable-search --syntax-module TEST
+KRUN_FLAGS+=--search-final --no-pattern
+
+include /home/robertorosmaninho/rv/k/k-distribution/include/kframework/ktest.mak
```

Comment:
This is an absolute path on your own machine; it needs to be a relative path pointing upwards to `ktest.mak`. Take a look at the other Makefiles in sibling test directories to get the right path.

Reply:
Lack of attention, sorry about it!